### PR TITLE
Fixed the error in conv1d example

### LINKED ISSUE #51225: [Docs] Example for torch.nn.functional.conv1d throws an error
## 📚 Documentation
The given example in https://pytorch.org/docs/stable/nn.functional.html#conv1d throws an error

```
>>> filters = torch.randn(33, 16, 3)
>>> inputs = torch.randn(20, 16, 50)
>>> F.conv1d(inputs, filters)
>>> RuntimeError: Calculated padded input size per channel: (3). Kernel size: (50). Kernel size can't be greater than actual input size
```

cc @jlin27 @mruberry @albanD @jbschlosser

### PR DESCRIPTION
**Pull request by harishsdev**
Fixes #51225

### CODE REVIEW

**Comment by harishsdev** [comment]
@gchanan please help to review

File: torch/nn/functional.py [hunk 1]
```diff
@@ -62,8 +62,8 @@
 
 Examples::
 
-    >>> filters = torch.randn(33, 16, 3)
-    >>> inputs = torch.randn(20, 16, 50)
+    >>> inputs = torch.randn(33,16,30)
```
Comment (albanD) [inline torch/nn/functional.py:65]: Could you add back the proper spacing after the `,` please?

**Review by albanD — COMMENTED** [review]
Thanks for the update!
Just a minor formatting update and this is good to go.

File: torch/nn/functional.py (hunk 1 above)
Comment (harishsdev) [reply torch/nn/functional.py]: @albanD  Thanks for reply,modification done

**Comment by malfet** [comment]
Hmm, I I can not reproduce an error reported in #51225:
```
$ python3 -c "import torch;filters = torch.randn(33, 16, 3); inputs = torch.randn(20, 16, 50); print(torch.__version__, torch.sum(torch.nn.functional.conv1d(inputs, filters)))"
1.8.1+cu102 tensor(124.8502)
```

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**